src/starknetCli.ts

```diff
@@ -210,18 +211,23 @@ export async function runStarknetCallOrInvoke(
     }
     throw e;
   }
-
   try {
-    execSync(
+    let warpOutput: string = execSync(
       `${warpVenvPrefix} starknet ${callOrInvoke}  --address ${options.address} --abi ${abiPath} --function ${funcName} --network ${options.network} ${wallet} ${account} ${inputs}`,
-      { stdio: 'inherit' },
-    );
+    ).toString('utf-8');
+
+    if (!options.use_cairo_abi) {
```

Comment:
can you add a check if this was a call, when invoking we don't get a result that we can decode